test-helpers/src/lib.rs

```diff
@@ -38,16 +38,73 @@ pub fn run_command(command: &str, args: &[&str]) -> Result<String> {
     }
 }
 
+fn command_not_found_error_message(command: &str, args: &[&str]) -> String {
+    // Maps a command to its associated dependency; however, if the name of the command and dependency is the same, we just use the command name.
+    // Currently, the only use case is mapping `npm` to its dependency `nodejs`.
+    let dependency = match command {
+        "npm" => "nodejs",
+        _ => command,
+    };
+
+    let args_part = if !args.is_empty() {
+        format!(" {}", args.join(" "))
+    } else {
+        String::new()
+    };
+
+    format!(
+        "Attempted to run the command `{}{}` but {} does not exist. Have you installed {}?",
+        command, args_part, command, dependency
+    )
+}
+
 pub async fn run_command_async(current_dir: &Path, command: &str, args: &[&str]) {
     let output = tokio::process::Command::new(command)
         .args(args)
         .current_dir(current_dir)
         .kill_on_drop(true)
         .status()
         .await
+        .map_err(|e| {
+            if e.kind() == std::io::ErrorKind::NotFound {
+                return Error::msg(command_not_found_error_message(command, args));
```

Comment:
https://docs.rs/anyhow/latest/anyhow/struct.Error.html#method.msg as per the docs on Error::msg, the `anyhow!` macro is generally preferred as it covers both the case of an error string and a wrapped error correctly.

It looks like you've manually used the function calls in the right place though, so I'm not going to block on this.

Reply:
Thank you for your all feedback! Appreciate it.